libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -926,64 +936,71 @@ void AC_PosControl::update_z_controller()
         }
     }
     _last_update_z_ticks = AP::scheduler().ticks32();
+    float thr_out;
+    if (use_z) {
```

Comment:
I believe this line, update_throttle and use_z_control are the only things I actually changed in this file, the rest is git getting confused